src/helpers/send_location_helper.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import os
+
+BASESTATION_URL = os.getenv("BASESTATION_URL", "https://basestation.example.com")
```

Comment:
This would not work, as the basestation url is not known and should be derived from the routing.